src/lib/__tests__/gloss.test.js

```diff
@@ -0,0 +1,111 @@
+import { GrabGlossProperties, processGloss } from "../utils";
+import axios from "axios";
+
+// Mock axios
+jest.mock("axios");
+
+describe("GrabGlossProperties", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("should send an appropriate RERUM query", async () => {
+    axios.post.mockResolvedValue({
+      data: { status: 200, someProperty: "someValue" },
+    });
+    const response = await GrabGlossProperties(
+      "https://store.rerum.io/v1/id/614362c3e74876243131a4e0",
+    );
+
+    expect(axios.post).toHaveBeenCalledWith(
+      "https://tinymatt.rerum.io/gloss/query?limit=100&skip=0",
+      {
+        $or: [
+          { target: "http://store.rerum.io/v1/id/614362c3e74876243131a4e0" },
```

Comment:
This is so messy. I think we can schedule some time to correct these targets when we feel like we have it closer to deliverable.